development/index.js

```diff
@@ -15,20 +15,18 @@ require (['bigscreenplayer/bigscreenplayer'], function(BigscreenPlayer){
   let minimalData = {
     initialPlaybackTime: 30,
     media: {
-      captionsUrl: '',
-        type: 'application/dash+xml',
-        mimeType: 'video/mp4',
-        kind: 'video',
-        urls: [
-          {
-            // Content from DASH IF testing assests (used in their reference player)
-            // https://reference.dashif.org/dash.js/v2.9.2/samples/dash-if-reference-player/index.htm
-            url: 'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd',
-            cdn: 'dash.akamaized.net'
-          }
-        ]
-      }
-    };
+      captions: [],
```

Comment:
Maybe not doable, but is there a supplied example vid that _does_ have subtitles we could use?

Reply:
I'll look into it!